server/api/batch_tallies.py

```diff
@@ -311,3 +320,57 @@ def download_batch_tallies_template(
         string_io,
         filename=f"candidate-totals-by-batch-template-{jurisdiction_timestamp_name(election, jurisdiction)}.csv",
     )
+
+
+@api.route(
+    "/election/<election_id>/batch-tallies/summed-by-jurisdiction-csv", methods=["GET"],
+)
+@restrict_access([UserType.AUDIT_ADMIN, UserType.JURISDICTION_ADMIN])
+def download_batch_tallies_summed_by_jurisdiction_csv(election: Election):
+    string_io = io.StringIO()
+    csv_writer = csv.writer(string_io)
+
+    contest_choice_csv_headers = construct_contest_choice_csv_headers(election)
+    csv_headers = [
+        "Jurisdiction",
+        *contest_choice_csv_headers.values(),
```

Comment:
Same note here as in https://github.com/votingworks/arlo/pull/1893#discussion_r1511328787 re dictionaries being ordered